README.md

```diff
@@ -11,6 +11,11 @@ Simply press a key (`<C-l>` in insert mode by default) and select what you want
 
 `insert_link` only works for elements in the current document.
 
+### Automatic File Link Insertion
+Simply press a key (`<C-f>` in insert mode by default) and select what Neorg file you want to link to.
```

Comment:
this isn't defined anywhere right now right?

Reply:
Nope, took a guess at the mapping desired mapping, is this set in neorg?

What would you preffer, taking this out and leaving it to neorg-telescope and nvim-neorg code owners to manage?

Reply:
@vhyrro